src/Compilers/Core/Portable/InternalUtilities/CompilerOptionParseUtilities.cs

```diff
@@ -20,13 +20,7 @@ public static IList<string> ParseFeatureFromMSBuild(string features)
             }
 
             var all = features.Split(new[] { ';', ',', ' ' }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
You'll have ask the original implementor, I'm just keeping the existing variable names.
There are other places in the source, where a "temp" is declared just before returning from a function. Think it help when debugging, to have a place where you can attach a breakpoint.